bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/discovery/ThingDiscoveryService.java

```diff
@@ -59,31 +66,22 @@ public class ThingDiscoveryService extends AbstractDiscoveryService implements T
             new AbstractMap.SimpleEntry<>("TWINGUARD", BoschSHCBindingConstants.THING_TYPE_TWINGUARD),
             new AbstractMap.SimpleEntry<>("PSM", BoschSHCBindingConstants.THING_TYPE_INWALL_SWITCH), // Steckdosen Zwischenstecker
```

Comment:
what does `PSM` mean?

Reply:
The whole list is reverse engineered, based on own data and on a good guess.

The [SHC API docu](https://local.apidocs.bosch-smarthome.com/#/Devices/get_devices) contains nothing. But in other SHC github projects like [boschshcpy](https://github.com/tschamm/boschshcpy/blob/master/boschshcpy/models_impl.py) you can find similar lists.

The ``PSM`` is the normal Smart Plug. I guess most the stuff is related to the HomeMatic IP system.